README.md

```diff
@@ -308,3 +308,132 @@ input SomeInput {
   flagsAreSupported: String! @pattern(regexp: "[a-z]+", flags: "i")
 }
 ```
+
+### Relay (Global) Node ID Support
+
+This package exposes two directives to convert IDs encode and decode Relay's
+[Node](https://facebook.github.io/relay/graphql/objectidentification.htm) interface.
+
```

Comment:
I think you can link to our other repository https://github.com/profusion/apollo-federation-node-gateway, such as:

> For instance, this plays well with https://github.com/profusion/apollo-federation-node-gateway that will collect all the types implementing the `Node` interface as integers so the encoded id is both small and avoid leaking internal details.

Eventually we could even split that part out of `apollo-federation-node-gateway` and leave the node-gateway to use it, as this could be useful even without federation.